examples/multi_script_CLI.py

```diff
@@ -1,97 +1,607 @@
+            menu_like()
+        elif ans=="3":
+            menu_comment()
+        elif ans=="4":
+            menu_unfollow()
+        elif ans=="5":
+            menu_block()
+        elif ans=="6":
+            menu_setting()
+        elif ans=="7":
+            bot.logout
+            sys.exit()
+        else:
+           print("\n Not Valid Choice Try again")
+
+def menu_follow():
+    ans=True
+    while ans:
+        print("""
+        1. Follow from hashtag
+        2. Follow followers
+        3. Follow following
+        4. Follow someone media likers
+        5. Main menu
+        """)
+        ans=input("how do you want to follow?").strip()
+
+        if ans=="1":
+            print("""
+            1.insert username
+            2.use username database
+            """)
+            if "1" in sys.stdin.readline():
+                user_id=input("who?").strip()
+            elif "2" in sys.stdin.readline():
+                user_id=random.choice(bot.read_list_from_file(users_file))
+            nlikes=input('how much like per account?')
+            bot.like_followers(user_id, nlikes=None)
+
+        elif ans=="2":
+            print("""
+            1.insert username
+            2.use username database
+            """)
+            if "1" in sys.stdin.readline():
+                user_id=input("who?").strip()
+            else:
+                user_id=random.choice(bot.read_list_from_file(users_file))
             bot.follow_followers(user_id)
-        elif ans == "7":
-            user_id = input("Who? ").strip()
+
+        elif ans=="3":
+            print("""
+            1.insert username
+            2.use username database
+            """)
+            if "1" in sys.stdin.readline():
+                user_id=input("who?").strip()
+            else:
+                user_id=random.choice(bot.read_list_from_file(users_file))
             bot.follow_following(user_id)
-        elif ans == "8":
+
+        elif ans=="4":
+            print("""
+            1.insert username
+            2.use username database
+            """)
+            if "1" in sys.stdin.readline():
+                user_id=input("who?").strip()
+            else:
+                user_id=random.choice(bot.read_list_from_file(users_file))
+            medias = bot.get_user_medias(user_id, filtration=False)
+            if len(medias):
+                likers = bot.get_media_likers(medias[0])
+                for liker in tqdm(likers):
+                    bot.follow(liker)
+
+        elif ans=="5":
+            menu()
+
+        else:
+            print("the number is not in the list you know")
+            menu_follow()
+
+def menu_like():
+    ans=True
+    while ans:
+        print("""
+        1. Like from hashtag
+        2. Like followers
+        3. Like following
+        4. Like last media likers
+        5. Like our timeline
+        6. Main menu
+        """)
+        ans=input("how do you want to like?").strip()
+
+        if ans=="1":
```

Comment:
fix PEP8: spaces between any binary methods: +, =, == etc. Loot at TRAVIS output in the pull request